lib/DialogflowAgentAssignedHandler.ts

```diff
@@ -76,6 +76,9 @@ export class DialogflowAgentAssignedClass {
 				} else if (customFields && customFields.errorSession === true && dialogflowSessionErrorEventName) {
 					console.log(ErrorLogs.DIALOGFLOW_ERROR_SESSION);
 					eventParams.name = dialogflowSessionErrorEventName;
+				} else if (customFields && customFields.agentDisconnected === true && dialogflowSessionErrorEventName) {
+					console.log(InfoLogs.LIVEAGENT_DISCONNECTED);
+					eventParams.name = dialogflowAgentUnavailableEventName;
```

Comment:
If this `agentDisconnected` makes confusion with `agentUnavailable` we can refactor customField name